### ISSUE TITLE
Form field missing a label: Bulkarax Field Mappings

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3078]
vpat.hykudemo.org
Version 7.1

Descriptive summary
Every form element should have a descriptive text label:
<img width="2038" height="967" alt="Image" src="https://github.com/user-attachments/assets/9383a8db-5e7e-4c8f-911b-672d10a5ab5a" />

Rationale
Accessibility issue remediation

Unclear labels make form-filling harder for everyone.

A label may already be visible on the page — but to be accessible, that label needs to be associated with the form element in the HTML. This helps visitors using screen readers to understand what information is required, while also making it easier for speech-input users to control the form element.

WCAG 2.2: [Opens in new tab:Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)
Siteimprove Alfa: [Opens in new tab:Technical documentation for rule SIA-R8](https://alfa.siteimprove.com/rules/sia-r8)

Expected behavior
The form element "textarea" should have a label

Actual behavior
The form element "textarea" has an understandable label

Steps to reproduce the behavior

- Open the page
- Use the SiteImprove browser extension
